pkg/server/k8s/converters.go

```diff
@@ -409,13 +409,13 @@ func ingressSpec(namespace, name string, vHosts []string) *k8s_extensions.Ingres
 		}
 	}
 
-	spec := k8s_extensions.IngressSpec{
+	spec := k8s_networking.IngressSpec{
 		Rules: rules,
 	}
 
-	return &k8s_extensions.Ingress{
+	return &k8s_networking.Ingress{
 		TypeMeta: metav1.TypeMeta{
-			APIVersion: "extensions/v1beta1",
+			APIVersion: "networking.k8s.io/v1beta1",
```

Comment:
I’m so glad I’m alive to see this change 🤣

Reply:
This will work only for Kubernetes v1.14+